pkg/oidcclient/login.go

```diff
@@ -420,7 +429,7 @@ func (h *handlerState) baseLogin() (*oidctypes.Token, error) {
 // and parse the authcode from the response. Exchange the authcode for tokens. Return the tokens or an error.
 func (h *handlerState) cliBasedAuth(authorizeOptions *[]oauth2.AuthCodeOption) (*oidctypes.Token, error) {
 	// Ask the user for their username and password, or get them from env vars.
-	username, password, err := h.getUsernameAndPassword()
+	username, password, err := h.getUsernameAndPassword(h.out)
```

Comment:
Is this a mistake, or was this needed for some testing reason perhaps? It seems strange for a function on `h` to take `h.out` as an argument, since `h` is the receiver and therefore already has access to `out` without taking it as a parameter.

Reply:
It was a mistake